includes/definitions/uhp.yaml

```diff
@@ -0,0 +1,11 @@
+os: uhp
+text: 'UHP Networks VSAT Terminal'
+type: wireless
+icon: uhp
+discovery:
+    -
+        sysObjectID: 
+            - .1.3.6.1.4.1.8000
+mib_dir: uhp
+poller_modules:
+    netstats: false
```

Comment:
Hi @loopodoopo 
Looks very good now. Just a question : Why did you disable netstats ? Does it fail with this device in any way ? If it is only because you don't need it, please leave it enabled in the YAML and disable it in your device config.

Reply:
These VSAT devices take a pretty long time to pol because of VSAT is slow... The UHP modems don't support netstats (tested), so disabling it saves 70 seconds in polling which is never available. I prefer to keep it disabled :)